go-controller/pkg/ovn/dns_name_resolver/dns_test.go

```diff
@@ -168,7 +147,7 @@ func TestAdd(t *testing.T) {
 			},
 			addressSetOpsHelper: []ovntest.TestifyMockHelper{
 				{
-					OnCallMethodName: "SetAddresses",
+					OnCallMethodName: "AddAddresses",
```

Comment:
why did the method change for the old implementation?

Reply:
While rebasing I had changed `SetAddresses` to `AddAddresses`. I have changed it back now.